src/contents/all/components/knot/Knot/Knot.tsx

```diff
@@ -23,6 +37,18 @@ const Knot = props => (
       >
         {props.knot.postedAt}
       </Moment>
+      <Popconfirm
+        title="Delete note?"
+        okText="Delete"
+        okType="default"
+        overlayClassName="deleteConfirm"
+        icon={null}
+        onConfirm={deleteKnot}
+      >
+        <span className={`hasAction ${noteHover || isMobile ? "show" : "hide"}`}>
```

Comment:
We'll want to get rid of these custom things like `isMobile` since we'll never be on mobile for this project. Also take a look at `Knot` in mainframe to see why and how we use `deleteLoading` and `noteHover`. Delete loading might not be necessary and we can just have one simple `loading` state variable to leverage for both deleting and editing.